src/tests/integration/test_foundry_prove.py

```diff
@@ -733,3 +733,68 @@ def test_foundry_refute_node(
 
     # Proof passes again
     assert_pass(test, single(prove_res_3))
+
```

Comment:
Consider moving this in a new `test_hevm_prove.py`.

Reply:
Done. I am also running the tests only with the booster since one of the tests was failing without `failure_info` in the legacy backend.